README.md

```diff
@@ -1,17 +1,44 @@
-# Google CloudEvents – Python
+# Google Events Library for Python
 
-This repository contains types for CloudEvents issued by Google.
+This library provides classes of common event types on Google Cloud Platform
+(GCP). At this moment the following types are available:
 
-## Prerequisites
+| Event Type  | Python Class | Description |
+| ------------- | ------------- | ------------- |
+| Cloud Pub/Sub Message Published Event  | MessagePublishedEvent | This event is triggered when a new Cloud Pub/Sub event is published to a topic. |
+| Cloud Audit Log Written Event  | AuditLogWrittenEvent | The event is triggered when a new Cloud Audit Log entry is written.  |
 
-- Python 3.5 (or higher)
+## Installation and Usage
 
-## Generate
+**Note**: This library requires Python 3.7+.
 
-Generate libraries:
+To install this package, run
 
-```sh
-./gen.sh
+``` sh
+pip install --upgrade google-events
+```
+
+To use an event class, see the snippet below:
+
+``` python
+from google.events import MessagePublishedEvent
+
+# Parses a Dict into an event
+# The Dict may be an argument in a background Cloud Function,
+# or the payload of an HTTP request received by a Cloud Run service with event
+# triggers.
+event = MessagePublishedEvent.from_dict(event_data)
+print(event.message)
 ```
 
-Observe the new files in `src/`.
\ No newline at end of file
+## Generation
+
+**Note**: Before generating the package, install `git`,
+[quicktype](https://quicktype.io/), and Python 3.7+.
```

Comment:
Can we list these as prerequisites with links?

- I don't think we need to state the git dependency.
- How do you install quickstype?

Reply:
The git dependency is removed. The installation instructions for quicktype is available on their page; it's just a global npm package from what I see.